frontend/src/components/bucket/BucketTable.vue

```diff
@@ -38,7 +38,7 @@ const showPermissions = async (bucketId: string, bucketName: string) => {
   permissionBucketName.value = bucketName;
 };
 
-const displayPermissionsIcon = (bucket: Bucket) => {
+const getUserCanManageBucket = (bucket: Bucket) => {
   return permissionStore.getIsBucketActionAllowed(bucket.bucketId, getUserId.value, Permissions.MANAGE );
 };
```

Comment:
Do we want these to be named as get*? They may collide with our naming convention for computed getters.